packages/components/tests/integration/components/hds/interactive/index-test.js

```diff
@@ -131,10 +130,12 @@ module('Integration | Component | hds/interactive/index', function (hooks) {
     assert.dom('button#test-interactive').hasAttribute('type', 'submit');
   });
   test('it should dispatch a click event when pressing space key on a link', async function (assert) {
-    await render(hbs`<Hds::Interactive @href="#" id="test-interactive"/>`);
-    var interactiveComponent = document.querySelector('#test-interactive');
-    var keyUpSpy = sinon.spy(interactiveComponent, 'click');
-    await triggerKeyEvent(interactiveComponent, 'keyup', ' ');
-    assert.true(keyUpSpy.calledOnce);
+    let clicked = false;
+    this.set('clickHandler', () => (clicked = true));
+    await render(
+      hbs`<div {{on "click" this.clickHandler}}><Hds::Interactive @href="javascript:;" id="test-interactive"/></div>`
```

Comment:
So we rely on the click event bubbling up, then capture it at the parent level using a boolean. Given we only use sinon for this test I'm happy with the change. We may get back to square one if we won't manage to use this pattern for future scenarios, but that's a problem for tomorrow.

Reply:
> We may get back to square one if we won't manage to use this pattern for future scenarios

That was my fear with "fixing" it this way as well.